include/rsl/parameter_validators.hpp

```diff
@@ -0,0 +1,301 @@
+#pragma once
+
+#include <rsl/algorithm.hpp>
+
+#include <rclcpp/rclcpp.hpp>
+#include <tl_expected/expected.hpp>
+
+#include <fmt/ranges.h>
+#include <string>
+#include <vector>
+
+namespace rsl {
+/**
+ * @cond DETAIL
+ */
```

Comment:
What does this do?

Reply:
This prevents doxygen from considering the lines between this and the closing one unless someone asks for DETAIL docs by defining that somehow.